### PR TITLE
display toolbar in screen configuration

### PR DESCRIPTION
**Pull request by lunika**
I can't remove the `Toolbar` component, if I do that I don't know how to fully display the `Content`. I think the `Content` componen itself is not made to be render without a toolbar.

### CODE REVIEW

**Comment by mikbry** [comment]
You need to set css for second content:
```
    position: fixed;
    top: 0;
    left: 0;
    height: 100%;
    width: 100%;
    border: 0px;
```
Also iframe has a border you to need clear it.

**Comment by lunika** [comment]
yes the border is a commit in opla/front I don't push for now

File: src/shared/containers/app.jsx [hunk 1]
```diff
@@ -278,7 +278,20 @@ class App extends React.Component {
...
+        <Content
+          style={
+            displayToolbar === false
+              ? {
+                  position: "fixed",
+                  top: 0,
+                  left: 0,
+                  height: "100%",
+                  width: "100%",
+                  border: "0px",
```
Comment (mikbry) [inline src/shared/containers/app.jsx]: Not sure the border is needed

File: src/shared/containers/app.jsx [hunk 2]
```diff
@@ -245,8 +245,8 @@ class App extends React.Component {
               {toolbox}
               <UserBox store={this.props.store} />
             </ToolbarRow>
-          )}
-        </Toolbar>
+          </Toolbar>
+        )}
         <Drawer
```
Comment (mikbry) [inline src/shared/containers/app.jsx]: The drawer could be put in above condition

Comment (lunika) [reply src/shared/containers/app.jsx]: if `displayToolbar` is set to `false` I also remove the `Drawer` ? Is it not too much ? We could want to display it another way than from the hamburger in the toolbar.

File: src/shared/containers/app.jsx [hunk 3]
```diff
@@ -161,6 +164,10 @@ class App extends React.Component {
     let toolbox;
     let fab;
     if (currentScreen) {
+      if (currentScreen.displayToolbar != null) {
```
Comment (mikbry) [inline src/shared/containers/app.jsx]: could be undefined instead of null ?

File: src/shared/containers/app.jsx (hunk 2 above)
Comment (mikbry) [reply src/shared/containers/app.jsx]: That why I created an issue with isFullScreen: true, to remove both ;-)

File: src/shared/containers/app.jsx (hunk 3 above)
Comment (lunika) [reply src/shared/containers/app.jsx]: yes this is why I used `!=` and not `!==`. Using `!=` will manage case where `displayToolbar` is null or undefined

https://jsfiddle.net/xomq737q/3/

File: src/shared/containers/app.jsx (hunk 1 above)
Comment (lunika) [reply src/shared/containers/app.jsx]: I removed it

File: src/shared/containers/app.jsx (hunk 2 above)
Comment (lunika) [reply src/shared/containers/app.jsx]: Where is this issue with the fullscreen ? Indeed calling it fullscreen is maybe a better idea and then we could hide Toolbar and Drawe

Comment (mikbry) [reply src/shared/containers/app.jsx]: https://github.com/Zoapp/front/issues/63

Comment (lunika) [reply src/shared/containers/app.jsx]: ok great I will use this parameter, I didn't saw this issue before.

**Comment by lunika** [comment]
@mikbry I finally change the parameter to call it `isFullscreen` and I hide both the toolbar and the drawer.

**Pull request merged**